### PR TITLE
Change checkbox state by click on checkbox label

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/jsdialog/Widget.TreeView.js [hunk 1]
```diff
@@ -80,23 +80,27 @@ function _createCheckbox(parentContainer, treeViewData, builder, entry) {
...
-				builder.callback('treeview', 'change', treeViewData, {row: entry.row, value: false}, builder);
-			}
+			_chnageCheckboxStateOnClick(this, treeViewData, builder, entry);
 		});
 	}
 
 	return checkbox;
 }
 
+function _chnageCheckboxStateOnClick(checkbox, treeViewData, builder, entry) {
```
Comment (hfiguiere) [inline browser/src/control/jsdialog/Widget.TreeView.js:90]: shouldn't this be `_changeCheckboxStateOnClick` (see typo) ?

Comment (Darshan-upadhyay1110) [reply browser/src/control/jsdialog/Widget.TreeView.js:90]: i see thanks for the save 🙇🏼

**Comment by eszkadev** [comment]
Could we do " Hide unfamiliar extra buttons in filter dialog " part in the core?
I don't like idea to add may hardcoded id's to hide/modify in Online, later it's hard to find what it is about, and it might be changed in the core and stop working.

In the core we can do:
`if (comphelper::LibreOfficeKit::isActive()) button->hide();`

**Comment by Darshan-upadhyay1110** [comment]
Sure @eszkadev let me change it in core. thanks :)

**Comment by Darshan-upadhyay1110** [comment]
> In the core we can do:
>if (comphelper::LibreOfficeKit::isActive()) button->hide();

https://gerrit.libreoffice.org/c/core/+/160082

CC: @eszkadev

**Review by eszkadev — APPROVED** [review]
looks good, please set PR to "ready for review"

**Pull request merged**